src/containers/FormikForm/VersionAndNotesPanel.jsx

```diff
@@ -0,0 +1,166 @@
+  AccordionPanel,
+  StyledAccordionsPanelItemsWrapper,
+  AccordionBar,
+} from '@ndla/accordion';
+import { VersionLogTag, VersionHistory } from '@ndla/editor';
+import FormikField from '../../components/FormikField';
+import * as draftApi from '../../modules/draft/draftApi';
+import { ArticleShape } from '../../shapes';
+import handleError from '../../util/handleError';
+import FormikAddNotes from './FormikAddNotes';
+import formatDate from '../../util/formatDate';
+import { fetchAuth0Users } from '../../modules/auth0/auth0Api';
+
+const paddingPanelStyleInside = css`
+  background: ${colors.brand.greyLightest};
+  padding: 0 ${spacing.normal};
+`;
+
+const getUser = (userId, allUsers) => {
+  const user = allUsers.find(user => user.id === userId) || {};
+  return user.name || '';
+};
+
+const getUsersFromNotes = async (notes, setUsers) => {
+  const userIds = notes
+    .map(note => note.user)
+    .filter(user => user !== 'System');
+  const uniqueUserIds = Array.from(new Set(userIds)).join(',');
+  const users = await fetchAuth0Users(uniqueUserIds);
+  const systemUser = { id: 'System', name: 'System' };
+  setUsers(
+    users && !users.error
+      ? [
+          ...users.map(user => ({
+            id: user.app_metadata.ndla_id,
+            name: user.name,
+          })),
+          systemUser,
+        ]
+      : [systemUser],
+  );
+};
+
+const VersionAndNotesPanel = ({ t, article }) => {
+  const [versions, setVersions] = useState([]);
+  const [users, setUsers] = useState([]);
+  useEffect(() => {
+    const getVersions = async () => {
+      try {
+        const versions = await draftApi.fetchDraftHistory(
+          article.id,
+          article.language,
+        );
+        setVersions(versions);
+      } catch (e) {
+        handleError(e);
+      }
+    };
+    getVersions();
+  }, [article]);
+
+  useEffect(() => {
+    if (versions.length) {
+      getUsersFromNotes(versions[0].notes, setUsers);
+    }
+  }, [versions]);
+
+  const cleanupNotes = notes =>
+    notes.map(note => ({
+      ...note,
+      author: getUser(note.user, users),
+      date: formatDate(note.timestamp),
+      status: t(`form.status.${note.status.current.toLowerCase()}`),
+    }));
+
+  return (
+    <Accordion openIndexes={[0]} tiny>
+      {({ getPanelProps, getBarProps }) => (
+        <AccordionWrapper>
+          {versions.map(({ revision, updated, published, notes }, index) => {
+            const current = index === 0;
+            return (
+              <Fragment key={revision}>
+                <AccordionBar {...getBarProps(index)} title={revision}>
+                  <StyledAccordionsPanelItemsWrapper>
+                    <div>{formatDate(updated)}</div>
+                    <div>
+                      {current && (
+                        <VersionLogTag color="yellow" label="Du er her" />
+                      )}
+                      {published && (
+                        <VersionLogTag color="green" label="Publisert" />
```

Comment:
Oversettelse? Og skal ikkje vises for current versjon.

Reply:
Hvorfor skal den ikke vises for nåværende versjon?